libraries/lib-wave-track/WaveClip.h

```diff
@@ -160,11 +160,13 @@ class WAVE_TRACK_API WaveClip final :
 
    // Set rate without resampling. This will change the length of the clip
    void SetRate(int rate);
+   void SetRawAudioTempo(double tempo);
 
    //! Stretches from left to the absolute time (if in expected range)
    void StretchLeftTo(double to);
    //! Sets from the right to the absolute time (if in expected range)
    void StretchRightTo(double to);
+   void StretchBy(double ratio);
```

Comment:
I really don't like it that lib-wave-track and lib-track have all this mention of stretches and tempos, which are extraneous to what should be lesser and more easily understood class responsibilities.

It's not for you to clean up now, but along with wave clip refactoring, I want to figure out dependency inversion between lib-wave-track and lib-stretching-sequence.